apps/condo/domains/common/components/icons/DownloadIcon.tsx

```diff
@@ -0,0 +1,21 @@
+import Icon from '@ant-design/icons'
+import React from 'react'
+
+interface DownloadProps {
+    width?: number | string
+    height?: number | string
+}
+
+export const DownloadIconSvg: React.FC<DownloadProps> = ({ width = 12, height = 8 }) => {
```

Comment:
This icon already exists at icon pack of the ui-kit 
![image](https://github.com/open-condo-software/condo/assets/25844927/3baf6248-f51e-40b0-8e59-8f7bdb835a2d)